syntax = "proto3";

service FacebookClone {
  rpc Register (RegisterRequest) returns (BasicReply) {}
}

message RegisterRequest {
  string userName = 1;
}

message BasicReply {
  bool success = 1;
  string reason = 2; // only needed for when success is false
}